### PR TITLE
Update the style guide to mention attributes

### PR DESCRIPTION
**Pull request by AndrejMitrovic**

### CODE REVIEW

File: doc/Style.md [hunk 1]
```diff
@@ -98,6 +98,43 @@ With parameters: `void foobar (int p1, void* p2)`.
 Templated functions are written as `void foobar (T) (T arg)`.
 This makes functions easy to `grep` for.
 
+### Attributes (@trusted, @safe, @nogc, pure, etc)
+
+Prefer adding `@safe`, `@nogc`, and `pure` atttributes to functions when it's
+possible. If one or more statements inside of the functions call a non-safe or
```
Comment (Geod24) [inline doc/Style.md]:
```suggestion
possible. If one or more statements inside a non-trivial function call a non-safe or
```

**Comment by AndrejMitrovic** [comment]
Fixed.

File: doc/Style.md [hunk 2]
```diff
@@ -98,6 +98,44 @@ With parameters: `void foobar (int p1, void* p2)`.
 Templated functions are written as `void foobar (T) (T arg)`.
 This makes functions easy to `grep` for.
 
+### Attributes (@trusted, @safe, @nogc, pure, etc)
+
+Prefer adding `@safe`, `@nogc`, and `pure` atttributes to functions when it's
+possible. If one or more statements inside of a non-trivial function calla a
```
Comment (Geod24) [inline doc/Style.md]:
```suggestion
possible. If one or more statements inside of a non-trivial function call a
```

**Comment by AndrejMitrovic** [comment]
Fixed typo.

**Comment by AndrejMitrovic** [comment]
calls => call fixed again heh

**Review by Geod24 — APPROVED** [review]

**Pull request merged**